app/components/user_item/user_item.tsx

```diff
@@ -1,196 +1,202 @@
 import {makeStyleSheetFromTheme, changeOpacity} from '@utils/theme';
 import {typography} from '@utils/typography';
-import {getUserCustomStatus, isBot, isCustomStatusExpired, isGuest, isShared} from '@utils/user';
+import {displayUsername, getUserCustomStatus, isBot, isCustomStatusExpired, isGuest, isShared} from '@utils/user';
 
 import type UserModel from '@typings/database/models/servers/user';
 
 type AtMentionItemProps = {
-    user?: UserProfile | UserModel;
-    containerStyle?: StyleProp<ViewStyle>;
+    user: UserProfile | UserModel;
     currentUserId: string;
-    showFullName: boolean;
     testID?: string;
     isCustomStatusEnabled: boolean;
-    pictureContainerStyle?: StyleProp<ViewStyle>;
+    showBadges?: boolean;
+    locale?: string;
+    teammateNameDisplay: string;
+    rightDecorator?: React.ReactNode;
+    onUserPress?: (user: UserProfile | UserModel) => void;
+    onUserLongPress?: (user: UserProfile | UserModel) => void;
+    disabled?: boolean;
+    viewRef?: React.LegacyRef<View>;
+    padding?: number;
 }
 
-const getName = (user: UserProfile | UserModel | undefined, showFullName: boolean, isCurrentUser: boolean, intl: IntlShape) => {
-    let name = '';
-    if (!user) {
-        return intl.formatMessage({id: 'channel_loader.someone', defaultMessage: 'Someone'});
-    }
-
-    const hasNickname = user.nickname.length > 0;
-
-    if (showFullName) {
-        const first = 'first_name' in user ? user.first_name : user.firstName;
-        const last = 'last_name' in user ? user.last_name : user.lastName;
-        name += `${first} ${last} `;
-    }
-
-    if (hasNickname && !isCurrentUser) {
-        name += name.length > 0 ? `(${user.nickname})` : user.nickname;
-    }
-
-    return name.trim();
-};
-
-const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
+const getThemedStyles = makeStyleSheetFromTheme((theme: Theme) => {
     return {
-        row: {
-            height: 40,
-            paddingVertical: 8,
-            paddingTop: 4,
-            paddingHorizontal: 16,
-            flexDirection: 'row',
-            alignItems: 'center',
-        },
-        rowPicture: {
-            marginRight: 10,
-            marginLeft: 2,
-            width: 24,
-            alignItems: 'center',
-            justifyContent: 'center',
-        },
-        rowInfo: {
-            flexDirection: 'row',
-            overflow: 'hidden',
-        },
         rowFullname: {
             ...typography('Body', 200),
             color: theme.centerChannelColor,
-            paddingLeft: 4,
+            flex: 0,
             flexShrink: 1,
         },
         rowUsername: {
-            ...typography('Body', 200),
+            ...typography('Body', 100),
             color: changeOpacity(theme.centerChannelColor, 0.64),
-            fontSize: 15,
-            fontFamily: 'OpenSans',
-        },
-        icon: {
-            marginLeft: 4,
         },
     };
 });
 
+const nonThemedStyles = StyleSheet.create({
```

Comment:
What's the difference between create and other methods to create style

Reply:
`makeStyleSheetFromTheme` uses a memoized version of the `theme` while this does not, but in this case, there are no colors that come from the theme, so there is no need to memoize it.